src/pages/Student/Student.css

```diff
@@ -0,0 +1,201 @@
+.student__container{
+    max-width:1200px;
+    margin: auto;
+}
+  .student__main{
+      display:flex;
+      justify-content: space-between;
+     padding-top: 70px;
+  }
+  
+  .student__info {
+    padding-top: 100px;
+    align:left;
+  }
+  
+  .student__h1 {
+    width: 40rem;
+    font-size: 6rem;
+    color:#141639;
+  }
+  
+  .student_p {
+    font-size: 1.50rem;
+    margin: 2rem 0;
+    font-weight: bold;
+    color: #73747b;
+  }
+  
+  .teacher1--img {
+    width: 590px;
+    height:470px;
+  }
+
+
+  .button1{
```

Comment:
avoid using directly "button" prefer to use more informative names like "teachers_login_btn"